jest.env.ts

```diff
@@ -21,7 +21,7 @@ import faucets from './.fuel-core/configs/faucets.json';
  * See: https://jestjs.io/docs/environment-variables#jest_worker_id
  */
 const getPrivateKeyForCurrentWorker = () => {
-  // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+   
```

Comment:
So you decided to allow _no-non-null_ assertions freely now?

Reply:
No, it seems like the default changed from version to version. I set it back to warning again.

Reply:
Gotcha. To be clear, I have no hard feelings about it.

These `!` assertions must be manually set _anyway_; we could just as well turn off this rule. 👍